Code/STAGE1bih_merging_and_organization.R

```diff
@@ -0,0 +1,49 @@
+## Jericho Lawson
+## Summer 2019, 2021
+## Merging and Organization
+
+# Adds calculated features to the data gathered from the MIT-BIH/2017 subjects.
+# Merges all data into one file.
+
+## Pick the correct version for testing. ##
+VERSION = "M"
+#VERSION = "C"
+#VERSION = "multi_2017"
+
+# Array of subjects.
+if (VERSION == "M"){
+  subjects = c('04015', '04043', '04048', '04126', '04746', '04908', '04936', '05091', '05121', 
+               '05261', '06426', '06453', '06995', '07162', '07859', '07879', '07910', '08215', 
+               '08219', '08378', '08405', '08434', '08455')
+}else{
+  subjects = read.csv("C:/Users/mario/Documents/Internship and REU Stuff/UNCW 2019/Atrial_Fibrillation/Testing/training2017/REFERENCE_2.csv", header = F)$V2[-1]
```

Comment:
We need to have a variable with the file path of data that the user can change. Then we just append the file name to it where we call it.